manifests/stage-rdma-device-plugin/0020-k8s-rdma-shared-dev-plugin-ds.yaml

```diff
@@ -48,6 +48,13 @@ spec:
         imagePullPolicy: IfNotPresent
         securityContext:
           privileged: true
+        env:
```

Comment:
We need to propagate these variables only to relevant stages.
e.g stages that their workload require internet connectivity. (e.g do `apt-get update`)

its not needed for this stage